setup.sh

```diff
@@ -81,6 +81,9 @@ STAGE_ENVIRONMENT_DIR=terraform/environments/staging
 export TF_VAR_project_id=${STAGE_PROJECT}
 export TF_VAR_ops_project_id=${OPS_PROJECT}
 terraform -chdir=${STAGE_ENVIRONMENT_DIR} init
+terraform -chdir=${STAGE_ENVIRONMENT_DIR} import \
+    module.emblem_staging.google_storage_bucket.sessions \
+    "${STAGE_PROJECT}-sessions" || true
```

Comment:
This doesn't seem like it should be in setup, but in some pre-setup process that's facilitating project reuse.

Reply:
AFAIK it needs to happen after `tf init`, but before `tf apply`.

**Idea:** should we make it _opt-in_ via an env var?

Reply:
Still confused why this is needed?

Reply:
This is needed _for project reuse_.

That said - I guess we could call `tf init` elsewhere first, and then call it here again?